### ISSUE TITLE
Add Frequently Asked Questions (FAQ) to contributing  guidelines/documentation

### ISSUE THREAD

**Issue by tatianamac** [tatianamac/selfdefined#246]
When should I open an issue versus a PR?

Open PR

- Correct misspellings
- Fix broken links
- Add definitions/nuance/translations to words that exist on the site already but are undefined/partially defined/could use more nuance
- Resolve issues that already exist 

Open Issue
- Concern or question around a term that you don't have clear prescriptive feedback around
- Adding a new term that is not yet listed (with or without definition)
- Adding/enhancing a feature that you're unsure is desired

**Comment by donnavitan** [comment]
Did you want to start an FAQ section under the "Development & Build" or "Documentation?

**Comment by mannybecerra** [comment]
This FAQ content ^ helped me decide on whether or not to create [PR 285](https://github.com/selfdefined/web-app/pull/285) - gracias!

**Comment by tatianamac** [comment]
@donnavitan Sorry for missing this comment! I think I'd honestly like a FAQ would serve both areas! In general I'd like even an FAQ PAGE that has frequently asked questions (selfishly, a lot of it is questions that I get asked via DMs, emails, etc could be addressed here).

I'm happy to provide more context here. Is this something you'd be willing to take on? @mannybecerra  Don't know if you'd be willing to help/lead as well since you found this helpful?

**Comment by mannybecerra** [comment]
> @donnavitan Sorry for missing this comment! I think I'd honestly like a FAQ would serve both areas! In general I'd like even an FAQ PAGE that has frequently asked questions (selfishly, a lot of it is questions that I get asked via DMs, emails, etc could be addressed here).
> 
> I'm happy to provide more context here. Is this something you'd be willing to take on? @mannybecerra Don't know if you'd be willing to help/lead as well since you found this helpful?

I'd be happy to assist here. @donnavitan I can follow your lead here with @tatianamac 's direction or in whatever other way y'all fancy to approach this. On standby...

**Comment by tatianamac** [comment]
@mannybecerra As I'm not positive if @donnavitan is around here, would you be open to opening a PR to address this? From there we can see if @donnavitan is available to contribute and provide more context/edit as they're able?

I think this way we'll keep things moving while respecting both Donna's participation if so chosen.

**Comment by mannybecerra** [comment]
Absolutely, agreed @tatianamac . I'll get on it and reach out with any Qs along the way. ¿Bueno?

**Comment by donnavitan** [comment]
@mannybecerra @tatianamac This is great! Thank you for moving forward. I'm still figuring this out and will contribute!

@tatianamac What's the best to way to collect the FAQ you've received? Without having you do much more work! Could we open a shared Google doc where you can drop the Qs and As, where we can help edit?

**Comment by mannybecerra** [comment]
> @mannybecerra @tatianamac This is great! Thank you for moving forward. I'm still figuring this out and will contribute!
> 
> @tatianamac What's the best to way to collect the FAQ you've received? Without having you do much more work! Could we open a shared Google doc where you can drop the Qs and As, where we can help edit?

@donnavitan hey now—you betcha, team effort! I'd love your [co-lead on curating FAQ](https://github.com/selfdefined/web-app/pull/288#issuecomment-683357085)s for the new page. I think a goal we should strive for, you nailed in your comment: keeping the labor/work on Tatiana's part low-friction. Google doc seems like a good place to centralize this type of dynamic content, especially when we consider extending future work with translators. I'll follow your lead
